sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/Crashes.java

```diff
@@ -752,10 +752,10 @@ ErrorReport buildErrorReport(ManagedErrorLog log) {
                     ErrorReport report = ErrorLogHelper.getErrorReportFromErrorLog(log, throwable);
                     mErrorReportCache.put(id, new ErrorLogReport(log, report));
                     return report;
-                } catch (ClassNotFoundException ignored) {
-                    AppCenterLog.error(LOG_TAG, "Cannot read throwable file " + file.getName(), ignored);
-                } catch (IOException ignored) {
-                    AppCenterLog.error(LOG_TAG, "Cannot access serialized throwable file " + file.getName(), ignored);
+                } catch (ClassNotFoundException | StackOverflowError e) {
```

Comment:
Should these exceptions really have the same message?  I think we can be more explicit here so we can investigate more precisely when for support issues.

Reply:
`e` is logged in `AppCenterLog.error(LOG_TAG, "Cannot read throwable file " + file.getName(), e)` so it should be fine to reuse the same message.

Reply:
I would say we can even merge the 2 catch remaining catch blocks as the difference between messages is not useful.